legacy-sdk/whirlpool/tests/integration/two_hop_swap.test.ts

```diff
@@ -829,6 +827,359 @@ describe("two-hop swap", () => {
     );
   });
 
+  describe("partial fill", () => {
```

Comment:
I think it's still possible to partial fill on two hop swaps in the following cases:

ExactOut, where sqrt_price_limit_two != 0, partial filled on second leg
ExactIn.  , where sqrt_price_limit_one != 0, partial filled on first leg

As long as the other amount thresholds values are sufficiently low/high

Should we add additional test cases here? (success + failure depending on the other_amount_threshold?)

Reply:
Good point!
Added 2 test cases for two_hop_swap v1 & v2.